joss_qmc/paper.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Quasi-Monte Carlo Methods in Python'
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Matt Haberland
+    affiliation: 3
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 3
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) has become the standard way of
+sampling random numbers in the scientific Python ecosystem.
+These methods are fast and reliable, and the results are repeatable when a
+random seed is provided. However, sampling in high dimensions produces a lot of
+gaps and clusters of points. In integration problems, classical methods have a 
+low convergence rate meaning that a large sample size is required.
+
+By construction, Quasi-Monte Carlo (QMC) methods provide efficient, determinist
+(or not) and quality generators that can advantageously replace traditional
+methods. This can be decisive when the sampling size is limited or strong
+reproducibility guarantee is required. Also QMC methods are known to have
+better convergence rate than traditional Monte Carlo sampling (used by NumPy).
+
+QMC methods where added to SciPy [@virtanen2020scipy] after an extensive review
+and discussion period [@scipy2021qmc] that lead to a great collaboration
+between SciPy's maintainers and renown researchers in the field.
+
+The following set of features is currently available:
+
+- Sobol' and Halton sequences (scrambled and unscrambled),
+- Latin Hypercube Sampling (centred, optimized on $C^2$, orthogonal),
+- Poisson disk,
+- Multinomial and multivariate normal,
+- Discrepancy measures ($C^2$, wrap around, star-$L_2$, mixed),
```

Comment:
Did you plan to add a citation?

Reply:
yep now we can optimize any sample, this is why I have a dedicated point for optimization.

For the citation I added a ref for QMC that could act as a cover all. Not sure we want to add too many ref otherwise.